androidlibrary_lib/src/main/res/values/theme.xml

```diff
@@ -1,20 +1,255 @@
-        <item name="android:scrollbarAlwaysDrawVerticalTrack">true</item>
-        <item name="android:scrollbarAlwaysDrawHorizontalTrack">true</item>
-        <item name="actionBarStyle">@style/OdkActionBar</item>
-        <item name="preferenceTheme">@style/PreferenceThemeOverlay.v14.Material</item>
-    </style>
-</resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
+
+   <!-- Contains Light Theme -->
+
+   <style name="OdkActionBar"
+       parent="Widget.MaterialComponents.Light.ActionBar.Solid">
+      <item name="displayOptions">useLogo|showHome|showTitle</item>
+      <item name="background">@color/colorPrimary</item>
+      <item name="android:textColor">@color/colorOnPrimary</item>
+      <item name="android:itemBackground">@color/colorOnPrimary</item>
+
+      <item name="logo">@drawable/app_logo</item>
+   </style>
+
+   <!-- Light Theme-->
+   <style name="Opendatakit" parent="Theme.MaterialComponents.Light.NoActionBar">
+      <item name="android:scrollbarThumbHorizontal">@drawable/thumb</item>
+      <item name="android:scrollbarThumbVertical">@drawable/thumb</item>
+      <item name="android:scrollbarSize">2.5mm</item>
+      <item name="android:scrollbarAlwaysDrawVerticalTrack">true</item>
+      <item name="android:scrollbarAlwaysDrawHorizontalTrack">true</item>
+      <item name="android:statusBarColor">@color/white</item>
+
+
+      <!-- # Keep code commented for error reference.
+      <item name="android:colorPrimary">@color/colorPrimary</item>
+      <item name="colorSecondary">@color/colorSecondary</item>
+      <item name="android:colorPrimaryDark">@color/colorPrimaryDark</item>
+      -->
+
+      <item name="android:colorPrimary">@color/colorPrimary</item>
+      <item name="colorPrimaryVariant">@color/colorPrimaryVariant</item>
+      <item name="colorSecondary">@color/colorSecondary</item>
+      <item name="colorSecondaryVariant">@color/colorSecondaryVariant</item>
+      <item name="android:colorBackground">@color/colorBackground</item>
+      <item name="android:colorPrimaryDark">@color/colorPrimaryDark</item>
+
+      <item name="colorOnPrimary">@color/colorOnPrimary</item>
+      <item name="colorOnSecondary">@color/colorOnSecondary</item>
+      <item name="colorOnSurface">@color/colorOnSurface</item>
+      <item name="colorOnError">@color/colorOnError</item>
+
+      <item name="android:windowLightStatusBar" tools:ignore="NewApi">true</item>
+
+      <item name="colorAccent">@color/blue_300</item>
+      <!--  <item name="actionBarStyle">@style/OdkActionBar</item> # Not needed since we're dealing with custom app bar-->
+      <item name="preferenceTheme">@style/PreferenceThemeOverlay.v14.Material</item>
+
+      <item name="buttonStyle">@style/ButtonStyle</item>
+
+   </style>
+
+
+   <!--Individual Stylings For Different Components -->
+
+
+   <!-- AlertDialog Theme -->
+   <style name="OdkAlertDialogStyle"  parent="@style/ThemeOverlay.MaterialComponents.MaterialAlertDialog">
```

Comment:
It would be better to include the "X"  so the style would be "OdkXAlertDialogStyle"

Reply:
Resolved